data/ecosystems/a/astar-network.toml

```diff
@@ -4,31 +4,45 @@ title = "Astar Network"
 sub_ecosystems = [
   "ACryptoS",
   "adaotop",
+  "AirGap",
   "ArthSwap",
   "Artzero",
   "AstridDao",
+  "Band Protocol",
   "Celer Network cBridge",
+  "Coinversation",
+  "Covalent",
   "JPY Coin",
   "Kodadot",
   "Multichain (Anyswap)",
   "O3 Swap",
   "Octopus Network",
   "Paras",
   "PolkaEx",
+  "Polkasafe",
   "Poly Network",
+  "Pyth Network",
   "Rubic",
   "SiO2",
   "Sirius Finance",
   "Starfish Finance",
   "Starlay Finance",
   "StellaSwap",
+  "Subquery",
```

Comment:
Can be removed, I've just added it as Secondary internally

Reply:
May ask what being added as Secondary means?

Reply:
We have this concept of Primary / Secondary (supported) which is modelled in our internal db but that we haven't yet added to this open source taxonomy
<img width="682" alt="image" src="https://github.com/electric-capital/crypto-ecosystems/assets/5723039/b627bf08-3651-4db8-bcf6-345b078244eb">

Reply:
got it, thanks!